lib/l10n/app_es.arb

```diff
@@ -36,5 +36,12 @@
   "importKeyImportUsingPrivateKeyActionLink": "Pulse aquí",
   "@importKeyImportUsingPrivateKeyActionLink": {},
   "importKeyImportUsingPrivateKeyDescription": "si desea importar usando su clave privada.",
-  "@importKeyImportUsingPrivateKeyDescription": {}
+  "@importKeyImportUsingPrivateKeyDescription": {},
+  "loginFirstTimeHere": "Primera vez aqui?",
+  "loginClaimInviteCodeButtonTitle": "Reclamar codigo de invitacion",
+  "loginAlreadyHaveAnAccount": "Ya tienes una cuenta de Seeds?",
+  "loginImportAccountButtonTitle": "Importar cuenta",
+  "loginRecoverAccountActionSegment1": "Perdiste tu llave?",
+  "loginRecoverAccountActionLink": " Recuperar ",
+  "loginRecoverAccountActionSegment2": "tu cuenta aqui"
 }
```

Comment:
@buildc0de dou you think we can use this notation to separete sections(screens) in .arb files?
https://github.com/flutter/flutter/issues/71729#issuecomment-852059076

Reply:
@RaulUrtecho Yeah, I saw this comment previously and considered it 🤓 I feel it would be superfluous because all screens already have the same prefix. For example, `onboarding`, `importKey`, `login`. And I think these prefixes clearly separate the keys into sections and make it easy to add new keys as needed.

Reply:
Ohh its true I didn't notice those prefixes. nice.